pkg/kubectl/resource/helper.go

```diff
@@ -63,7 +64,7 @@ func (m *Helper) Get(namespace, name string, export bool) (runtime.Object, error
 	return req.Do().Get()
 }
 
-func (m *Helper) List(namespace, apiVersion string, export bool, options *metav1.ListOptions) (runtime.Object, error) {
+func (m *Helper) List(namespace, apiVersion string, export, requestTable bool, options *metav1.ListOptions) (runtime.Object, error) {
```

Comment:
You'll need this for `Get` too.

Reply:
Thanks, wired it through to `Get` as well

Reply:
No.  Yuck.  This should be a separate flag on Helper that changes the request type.  We'll have other things we change to, like PartialObjectMetadata.

You also shouldn't have to be parsing into things, you just need to add the table type to the scheme

Reply:
See https://github.com/kubernetes/kubernetes/pull/58404/commits/c672a854b7f58e80e6a4f7b1b05d51c695fb2a76#diff-92d4543eda898d1cb24544d3399bebb3R41 and https://github.com/kubernetes/kubernetes/pull/58404/commits/c672a854b7f58e80e6a4f7b1b05d51c695fb2a76#diff-1bc8485222e19c02f9aac7ecbc1cdfd6R50 for adding to the scheme

Reply:
Regarding setting this, we should be transforming request like https://github.com/kubernetes/kubernetes/pull/58404/commits/c672a854b7f58e80e6a4f7b1b05d51c695fb2a76#diff-b5780bd984478baab9841efd2c817719R267

And then handling the presence of a Table gracefully in the returned get.go code.